Select.js

```diff
@@ -290,10 +291,11 @@ define([
 			}
 		},
 		
-		getIdentity: function (dataItem) {
-			// Override of delite/Selection's method
-			return this.store.getIdentity(dataItem);
-		},
+		getIdentity: dcl.superCall(function (sup) {
+            return function (dataItem) {
+                return sup.call(this, dataItem);
```

Comment:
The indentation is fine in my file so jshint isn't raising errors.
But I think it is a problem with my tabs which are not well interprated by github. I'll fix it.